src/azure-cli/azure/cli/command_modules/role/_help.py

```diff
@@ -409,9 +409,7 @@
   - name: --scopes
     short-summary: >
         Space-separated list of scopes the service principal's role assignment applies to.
-        Starting from Azure CLI 2.35.0, --scopes argument will become required for creating role assignments.
-        Defaults to the root of the current subscription. e.g., /subscriptions/0b1f6471-1bf0-4dda-aec3-111122223333,
-        /subscriptions/0b1f6471-1bf0-4dda-aec3-111122223333/resourceGroups/myGroup, or
+        e.g., /subscriptions/0b1f6471-1bf0-4dda-aec3-111122223333/resourceGroups/myGroup,
```

Comment:
Subscription level role assignment should not be recommended, so removed from the help.